FrostyCmd/Profiles/Fifa20.json

```diff
@@ -0,0 +1,40 @@
+{
+  "Name": "FIFA20",
+  "DisplayName": "FIFA 20",
+  "InternalName": "fifa20",
+  "Deobfuscator": "Signature",
+  "TypeInfoSignature": "",
+  "DataVersion": 20190927,
+  "FrostbiteVersion": "3.0",
+  "HasStrippedTypeNames": false,
+  "Sources": [
+    {
+      "Path": "Patch",
+      "SubDirs": false
+    },
+    {
+      "Path": "Update",
+      "SubDirs": true
+    },
+    {
+      "Path": "Data",
+      "SubDirs": false
+    }
+  ],
+  "EbxVersion": 5,
+  "RequiresKey": true,
+  "MustAddChunks": false,
+  "EnableExecution": true,
+  "HasAntiCheat": false,
+  "EbxCompression": 15,
+  "ResCompression": 15,
+  "ChunkCompression": 15,

```

Comment:
newer fifa games only use zstd for ebx not res and chunks, so fifa 20 and up i believe